me.marzeq.xbox-cloud-gaming-electron.yaml

```diff
@@ -0,0 +1,58 @@
+app-id: me.marzeq.xbox-cloud-gaming-electron
+branch: stable
```

Comment:
```suggestion
```

Reply:
Why is that?

Reply:
you don't set the branch. Whatever example you used as model is incorrect.